### ISSUE TITLE
Control bar looks weird

### ISSUE THREAD

**Issue by vincePeKaWay** [AirenSoft/OvenPlayer#185]
Hey,

so I am new to this player, copied the master branch and set it up on a simple html website.
I am trying to display a livestream and the streaming works fine, I just can't get the control bar to look like in the demo.
It just looks like this, with the svg's being displayed in the wrong position.

![ovp_controlbar](https://user-images.githubusercontent.com/77489211/116923773-56017380-ac57-11eb-87c7-3766e3592a30.png)

Can you guide me on how to make it look like in the demo? Tried playing around with the stylesheet, but nothing seems to work...
Thanks!

**Comment by SangwonOh** [comment]
@vincePeKaWay Hi. Tell us about your browser environment. And if you are using other css in your html page, can you send us the full css?

**Comment by vincePeKaWay** [comment]
@SangwonOh That's what I also figured after opening this issue, that the stylesheet of the html template interferes with the stylesheet of OvenPlayer.
Anyway, I then used the starter template from the OvenPlayer Quick Start guide with no additional css files and the buttons had a white background. This can be easily overriden by setting the background of .op-button to transparent. Then it looks just like in the demo.

FYI: 
This is the css file from the template I used, it's rather old by now but I still use it for testing stuff from time to time:
https://github.com/ms-studio/templated-broadcast/blob/master/assets/css/main.css

I didn't check back where exactly the problem occurs though.

Thank you very much for your input!
